src/renderer/auth/Cloud.js

```diff
@@ -54,9 +55,8 @@ export class AuthAccess {
     // IDP client associated with current tokens; undefined if unknown; null if not specified
     idpClientId: [rtv.OPTIONAL, rtv.STRING],
 
-    // true if access is via SSO; false if it's via basic auth (which would require a
-    //  password, and which we no longer support); null if it's undetermined
-    usesSso: [rtv.OPTIONAL, rtv.BOOLEAN],
+    // URL to the MCC instance
+    cloudUrl: [rtv.OPTIONAL, rtv.STRING],
```

Comment:
One last thing I just noticed: This should be
```suggestion
    cloudUrl: [rtv.EXPECTED, rtv.STRING],
```
because the property is no longer optional. We want all Cloud instances to have a `cloudUrl`. But then you need to be able to `new Cloud()` to create one where `cloudUrl` will be null, initially, so in terms of [RTV.js](https://rtvjs.stefcameron.com/) validations, that means the value is `EXPECTED`. (RTV.js is a runtime validation library I wrote.)

Reply:
fixed